.circleci/config.yml

```diff
@@ -111,28 +111,28 @@ workflows:
   version: 2
   build:
     jobs:
-      - confirm_config
+      - confirm_config_and_documentation
 
-      # Use `requires: [confirm_config]` to trick Circle CI into starting the slow jruby job early.
+      # Use `requires: [confirm_config_and_documentation]` to trick Circle CI into starting the slow jruby job early.
```

Comment:
I noticed how this build can be incredibly slow now, e.g. https://circleci.com/gh/rubocop-rspec/rubocop-rspec/1982

Perhaps we should use `required: [code-climate]` instead for this “trick”?

Reply:
Ah, I found the issue. See comment on the rake task.

Reply:
> Perhaps we should use required: [code-climate] instead for this “trick”?

Just to be clear: Please disregard that suggestion now. :-)